### DISCUSSION TITLE
origin-top-right with screen selector is not working properly

### DISCUSSION THREAD

**Discussion by teddybee** [tailwindlabs/tailwindcss#10158] [Help] [answered]
I would like to flip a menu from left side to the right when the screen is small. Looks like there is a bug,  the selector does nothing, all screen looks the same.
```
origin-top-left left-0 md:origin-top-right md:right-0"
```
https://play.tailwindcss.com/O0VMoeu4eV?size=948x925

**Comment by wongjn** [comment, marked as answer]
`origin-*` utilities apply `transform-origin` property values. This would indeed have no effect on the element that you have applied it to. This works as expected.

You can move the menu by resetting the `left` value from `0` to `auto` instead: https://play.tailwindcss.com/vOiAizsKts
